packages/round-manager/src/features/round/ViewRoundSettings.tsx

```diff
@@ -1825,14 +1825,15 @@ function Funding(props: {
           </div>
 
           <div
-            className={`leading-8 font-normal ${
+            className={`leading-8 flex font-normal ${
               !props.editMode.canEdit ||
               (props.editMode.canEditOnlyRoundEndDate && "text-grey-400")
             }`}
           >
             <input
+              size={matchingFundPayoutToken.name.length ?? 3}
```

Comment:
If I'm understanding this correctly, and name is a string, then length will never be null or undefined, and the `?? 3` branch will never trigger.

Reply:
I think if name is empty (`""`) it will default to size 3. What happens to the input field when changing between empty, to one character, to two? Will it jump from size 3 to 1 and then 2?

Reply:
```
$ node
Welcome to Node.js v20.6.1.
Type ".help" for more information.
> "".length ?? 3
0
```

If name is empty, length is zero, which is neither null nor undefined, so the `??  3` branch won't trigger.

Reply:
the string will never be empty if the token is found, but the token could theoretically not be found for whatever reason, even though typescript doesn't capture this (because it assumes that .filter() on an array always produces a non-empty array), and therefore will be undefined. I forgot some opt. chaining operators, added in the latest commit.